### PR TITLE
Refactor serviceData (remove KeyValueProperty)

### PR DESCRIPTION
**Pull request by RubenMattan**
```python
def migrate_servicedata():
  from rogerthat.migrations.migrate_service_data import job
  
  job()
```

### CODE REVIEW

File: src/rogerthat/migrations/migrate_service_data.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    def trans():
+        si = db.get(si_key)
+        if si.appData:
+            return
+        if not si.serviceData:
+            logging.debug("data empty service_identity_user:%s" % (si.service_identity_user))
+            raise Exception("migrate_service_data failed")
+        
+        data_dict = si.serviceData.to_json_dict()
+        si.serviceData.clear()
```
Comment (lucasvanhalst) [inline src/rogerthat/migrations/migrate_service_data.py:47]: maybe add `si.serviceData = None` as well

File: src/rogerthat/to/service.py [hunk 2]
```diff
@@ -210,11 +210,11 @@ def fromServiceIdentity(service_identity, service_profile):
         details.admin_emails = [] if service_identity.metaData is None else [e.strip() for e in
                                                                              service_identity.metaData.split(',') if
                                                                              e.strip()]
-        if service_identity.serviceData:
+        if service_identity.appData:
+            details.app_data = service_identity.appData
+        else:
             service_data = service_identity.serviceData.to_json_dict()
```
Comment (lucasvanhalst) [inline src/rogerthat/to/service.py:216]: remove in followup pr

**Review by lucasvanhalst — CHANGES_REQUESTED** [review]
update property:
`appData = db.TextProperty()  # deprecated, lazily migrated to serviceData when putting service_data`
remove the comment & remove `serviceData` property in followup pr

File: src/rogerthat/migrations/migrate_service_data.py (hunk 1 above)
Comment (RubenMattan) [reply src/rogerthat/migrations/migrate_service_data.py:47]: needed to clear the buckets

**Pull request merged**